Src/Plugins/SalModernization/SalModernization.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\Targets\build.common.props" />
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\Targets\build.product.props" />
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\Targets\build.plugins.props" />
+  
+  <PropertyGroup Label="AssemblyAttributes">
+    <TargetFrameworks>netstandard2.0;net472</TargetFrameworks>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <RootNamespace>$(RootNamespaceBase).Sarif.PatternMatcher.Plugins.SalModernization</RootNamespace>
+    <PackageId>Sarif.PatternMatcher.SalModernization</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="SalModernization.SharedStrings.txt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="SEC105.UpdateSalToCurrentVersion.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Sarif.PatternMatcher.Sdk\Sarif.PatternMatcher.Sdk.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+	<SpamFiles Include="$(MSBuildThisFileDirectory)*.json" />
+  </ItemGroup>
```

Comment:
you can remove this

Reply:
Do you mean the two "remove" statements that follow this line?  If I remove the above, we don't get the JSON file in the .spam folder, as desired.

---
In reply to: [605121306](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605121306) [](ancestors = 605121306)

Reply:
take a look at this: [https://github.com/microsoft/sarif-pattern-matcher/blob/dc4850a8aa2b0b65cfc684eb007dbb4d536104c1/Src/Plugins/Security/Security.csproj#L58-L70](https://github.com/microsoft/sarif-pattern-matcher/blob/dc4850a8aa2b0b65cfc684eb007dbb4d536104c1/Src/Plugins/Security/Security.csproj#L58-L70)

if you see, i didnt create 3 item groups with the json.

---
In reply to: [605139713](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605139713) [](ancestors = 605139713,605121306)

Reply:
Right, I think we're saying the same thing.  The line that is highlighted for this comment is the bottom of the "SpamFiles Include" itemgroup.  I think you're asking me to remove the two itemgroups that follow that, right?

---
In reply to: [605141988](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605141988) [](ancestors = 605141988,605139713,605121306)

Reply:
you  could delete this and the follow, move the third to inside the targets tag to be similar to what i did in the security package.

---
In reply to: [605148884](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r605148884) [](ancestors = 605148884,605141988,605139713,605121306)